src/config/backends/odmdata.ts

```diff
@@ -127,16 +127,12 @@ export class Odmdata extends ConfigBackend {
 			);
 		}
 		// Find the configuration given the optionsBuffer
-		const configIndex = this.CONFIG_BYTES.reduce(
-			(currentIndex: number, _config: number, index: number) => {
-				if (
+		const configIndex = this.CONFIG_BYTES.findIndex(
```

Comment:
The previous behavior was technically `findLastIndex` which requires nodejs 18 but since in practice the check can only succeed for one index `findIndex` is equivalent. Also the benefit to the `find*` calls is that they short-circuit which means that they'll stop iterating the moment a match is found vs `reduce` which will iterate them all (in this case there's only 3 elements, but as a general rule being both clearer and more performant is a win)